src/NUnitEngine/nunit.engine.api/RuntimeType.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace NUnit.Engine
+{
+    /// <summary>
+    /// Enumeration identifying a common language
+    /// runtime implementation.
+    /// </summary>
+    public enum RuntimeType
+    {
+        /// <summary>Any supported runtime framework</summary>
+        Any,
+        /// <summary>Microsoft .NET Framework</summary>
+        Net,
+        /// <summary>Microsoft .NET Compact Framework</summary>
+        NetCF,
+        /// <summary>Microsoft Shared Source CLI</summary>
+        SSCLI,
+        /// <summary>Mono</summary>
+        Mono,
+        /// <summary>Silverlight</summary>
+        Silverlight,
+        /// <summary>MonoTouch</summary>
+        MonoTouch
```

Comment:
Exposing this bothers me. I don't think we set it and we don't/can't detect it, so should we be exposing it? It also begs the question as to if we are exposing this, SL and CF, why are we not exposing all of the other platforms that we support like varios WinPhone versions, Win8.1, .NET Core, Android, Xamarin.iOS, etc? My assumption would be that this list is limited to runtimes that the engine supports, not runtimes that the framework supports.